pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -615,7 +615,8 @@ private boolean processStreamEvents(MessageBatch messagesAndOffsets, long idlePi
       }
     } else if (!prematureExit) {
       // Record Pinot ingestion delay as zero since we are up-to-date and no new events
-      _realtimeTableDataManager.updateIngestionDelay(System.currentTimeMillis(), _partitionGroupId);
+      _realtimeTableDataManager.updateIngestionDelay(System.currentTimeMillis(), System.currentTimeMillis(),
```

Comment:
Refactor to `long now = System.currentTimeMillis(); ...`

Reply:
Addressed in commit 9bb0befe9a55c01bb3a5066a4b30137469819f38